src/services/UserService.ts

```diff
@@ -63,9 +63,11 @@ export class UserService {
         }
     }
 
-    public update(username: string, user: User): Promise<User> {
```

Comment:
I think the way it was previously was better because this change would require you to edit this function if the user model is updated